cachito/web/models.py

```diff
@@ -101,6 +102,36 @@ def get_final_states():
         """
         return ["complete", "failed"]
 
+    @staticmethod
+    def allow_transition(from_state: Optional[int], to_state: int) -> bool:
+        """Check if the state transition is allowed.
+
+        :param from_state: transition from this state.
+        :type from_state: int or None
+        :param int to_state: transition to this state.
+        :return: True if the transition is allowed, otherwise False is returned.
+        :rtype: bool
+        """
+        if from_state is None:
+            return True  # Allow to set the initial state.
+        allowed_from_states: Optional[List[int]] = AllowedRequestStateTransition.get(to_state)
+        if allowed_from_states is None:
+            return False
+        return from_state in allowed_from_states
+
+
+# to_state: from_states
+AllowedRequestStateTransition: Dict[int, List[int]] = {
```

Comment:
Why is the name in `CamelCase`? As a constant, shouldn't it be `UPPERCASE_WITH_UNDERSCORES`?